auth/src/main/res/values/strings.xml

```diff
@@ -100,4 +105,8 @@
     <string name="fui_verify_phone_number" translation_description="Button text to submit confirmation code and complete phone verification">Verify Phone Number</string>
     <string name="fui_continue_phone_login" translation_description="Button text to submit phone number and send sms">Continue</string>
     <string name="fui_sms_terms_of_service" translation_description="Fine print warning displayed below Verify Phone Number button">By tapping “%1$s”, an SMS may be sent. Message &amp; data rates may apply.</string>
+    <string name="fui_sms_terms_of_service_and_privacy_policy_extended" translation_description="Fine print warning displayed below Verify Phone Number button">By tapping “%1$s”, you are indicating that you accept our <xliff:g example="https://google.com/terms" id="tos" translation_description="">%2$s</xliff:g> and <xliff:g example="https://google.com/privacy" id="pp" translation_description="">%3$s</xliff:g>. An SMS may be sent. Message &amp; data rates may apply.</string>
```

Comment:
We're going to need to kill `fui_create_account_preamble_tos_and_pp` and the others to make lint happy. In the meantime, add `'MissingTranslation' // TODO remove in future PR` [here](https://github.com/SUPERCILEX/FirebaseUI-Android/blob/4e51a457b12e622604e61b473079b2107a54cfd7/library/quality/quality.gradle#L76-L79) and then @samtstern will probably have you create a second PR that kills those strings.

Reply:
Sent off the strings for translation.